docs/testing/debug-add-ins-overview.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Debug Office Add-ins
+description: 'Find the Office Add-in debugging guidance for your development environment'
+ms.date: 12/02/2021
+ms.localizationpriority: high
+---
+
+# Overview of debugging Office Add-ins
+
+Debugging Office Add-ins is essentially the same as debugging any web application; but there isn't a single set of tools that all add-in developers can use. This is because add-ins can be developed on different operating systems and can run cross-platform. This article will help you find the detailed debugging guidance for your development environment.
+
+> [!NOTE]
+> This article is concerned with debugging in the narrow sense of setting breakpoints and stepping through code. For guidance on testing and troubleshooting, start with [Test Office Add-ins](test-debug-office-add-ins.md) and [Troubleshoot development errors with Office Add-ins](troubleshoot-development-errors.md).
+>
+> This article uses "your development computer" and "your development environment" to refer to the environment on which you're debugging. This almost always *is* your development computer. Although you should *test* your add-in on all the platforms that you want to support, you'll only very rarely need to *debug* on an environment different from your development computer.  
+
+## Server-side or client-side?
+
+Debugging the server-side code of an Office Add-in is exactly the same as debugging the server-side of any web application, so there is no need for special documentation. See the debugging instructions for your IDE or other tools. The following are examples and you can find much more on the web.
+
+- [Debug ASP.NET or ASP.NET Core apps in Visual Studio](/visualstudio/debugger/how-to-enable-debugging-for-aspnet-applications)
+- [Debugging Express](https://expressjs.com/en/guide/debugging.html)
+- [Node.js Debugging Guide](https://nodejs.org/en/docs/guides/debugging-getting-started/)
+- [Node.js debugging in VS Code](https://code.visualstudio.com/docs/nodejs/nodejs-debugging)
+- [Webpack Debugging](https://webpack.js.org/contribute/debugging/)
+
+The rest of this article is concerned only with debugging client-side JavaScript (which may be transpiled from TypeScript).
+
+To find guidance for debugging client-side code, the first variable is the OS of your development computer.
```

Comment:
Since this is the only time "OS" is used in this doc, consider spelling it out.
```suggestion
To find guidance for debugging client-side code, the first variable is the operating system of your development computer.
```